docs/src/manual/variables.md

```diff
@@ -1082,13 +1082,13 @@ julia> model[:x]
 
 ## Semidefinite variables
 
-Declare a square matrix of JuMP variables to be positive semidefinite by passing
```

Comment:
We could just say here that there are the two possibilities at least, we should reopen and merge part of the changes of this PR.